views/partials/header.erb

```diff
@@ -21,7 +21,6 @@
       <ul class="nav">
         <li class="nav-link"><a href="/new">New Project?</a></li>
```

Comment:
This link currently goes nowhere. I propose we add the directions to https://github.com/18F/concourse-compliance-testing/blob/master/README.md and link there instead of a new page.